### ISSUE TITLE
FetchError: network timeout at: https://instancename.us1.cloud.realm.io/auth

### ISSUE THREAD

**Issue by razamerchantwork** [realm/realm-js#2576]
Hi,
We are continuously getting this error from Realm Cloud Server. When it occurs we are unable to receive realm time messages to our admin portal from mobile app. Is there any issues going in Realm cloud ? 

Here is the error stack trace:
```
{ FetchError: network timeout at: https://instancename.us1.cloud.realm.io/auth
 at Timeout._onTimeout (/APP_PATH/node_modules/node-fetch/index.js:126:13)
at ontimeout (timers.js:436:11)
at tryOnTimeout (timers.js:300:5)
at listOnTimeout (timers.js:263:5)
 at Timer.processTimers (timers.js:223:10)
name: 'FetchError',
message:
'network timeout at: https://instancename.us1.cloud.realm.io/auth',
type: 'request-timeout' }
```

## Version of Realm and Tooling

- Realm JS SDK Version: 2.20.0
- Node: 10.13.0
- Client OS & Version: Ubuntu 16.04.1 LTS

**Comment by razamerchantwork** [comment]
Is anyone maintaining this project ? 2 days have passed still no reply ?

**Comment by kneth** [comment]
@razamerchantwork Sorry for the late reply. For Realm Cloud issues, please create a ticket at https://support.realm.io/support/home.

We have fixed many bugs since Realm JS v2.20.0, so it might be a good idea to upgrade.

**Comment by kneth** [comment]
Closing for now. If problem persists, please reopen.
